docs/value(s).md

```diff
@@ -0,0 +1,21 @@
+# all about value(s)
```

Comment:
Should this be capitalized?

Reply:
no, should stay small

Reply:
or rather, capitalising the first letter - All about value(s)